docs/user/exams/instructors_guide.rst

```diff
@@ -0,0 +1,192 @@
+
+   Student Exam Page
+
+Assess Student Exams and Complaints
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- Once the :ref:`exam conduction <exam_conduction>` is over and the latest individual ``working time`` has passed, your team can begin the :ref:`assessment <exam_assessment>` process. 
+- This is done through the *Tutor Exam Dashboard*. 
+
+    - **Hint:** If the :ref:`exam conduction <exam_conduction>` is not over, you will not be able to access this page.
+
+- The :ref:`assessment <exam_assessment>` process is anonymised. All student information will not be displayed to the assessors.
+- The ´*Tutor Exam dashboard* provides an overview over the current assessment progress per exercise. For each exercise, you can view how many submissions have already been assessed and how many are still left. The status of the student complaints is also displayed here. 
+
+    - **Hint:** To check for plagiarism, you must navigate to the individual exercise. This can be done by navigating to:
+
+     *Exam Management* -> *Exercise Groups* -> *View* on the specific exercise.
+
+     At the bottom of the page you will find the option ``check for plagiarism``.
+
+.. figure:: instructor/tutor_dashboard.png
+   :alt: Tutor Exam Dashboard
+   :align: center
+
+   Tutor Exam Dashboard
+
+- To assess a submission for an exercise, you can click on ``Exercise Dashboard``.
+- First you must go through the example submissions and assessments to review how a specific exercise should be evaluated. 
+- If there is a submission which has not been assessed yet, you can click ``Start new assessment``. This will fetch a random student submission of this exercise which you can then assess.
+- Programming exercises are graded automatically but if ``manual assessment`` is allowed, see `Add Exercises`_, you can review and enhance the automatic results. 
+
+.. figure:: instructor/programming_assessment.png
+   :alt: Programming Submission Assessment
+   :align: center
+
+   Manually Assessing a Programming Submission
+
+- Quiz exercises are graded automatically via the student exam page, see `Manage Student Exams`_, and therefore do not appear in the *Tutor Exam Dashboard*.
+- Students can complain about their results during the :ref:`student review <student_review>` period. You can evaluate these complaints also in the *Tutor Exam Dashboard*. 
+- The original assessor may not respond to the complaint about his assessment, this must be done by a second assessor. 
+
+Exam Scores
+^^^^^^^^^^^
+- You can view the exam scores from the *Scores* page. This view aggregates the results of the students and combines them to provide an overview over the students’ performance. 
+- You can view the spread between different achieved scores, the average results per exercise as well as the individual students' results.
+- Additionally, you can choose to modify the dataset by selecting ``only include submitted exams`` or ``only include exercises with at least one non-empty submission``.
```

Comment:
shorty explain what those options mean and that they help in evaluating the statistics more realistically